src/Model/Consignment/AbstractConsignment.php

```diff
@@ -586,7 +586,7 @@ public function setStatus($status): self
     /**
      * @return integer|null
      */
-    public function getShopId(): int
+    public function getShopId(): ?int
```

Comment:
@RichardPerdaan in @blieb situation, the shop id is present. Do you think we should allow shop id null or not? If we don't allow it, we'll be more likely to notice that something unexpected is happening (like the above bug in ConsignmentAdapter.php). For getPickupStreet, getPickupCity and getPickupNumber it is valid to return null.